backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/acct/api/impl/AccountDAO.java

```diff
@@ -179,4 +184,29 @@ public AccountId getOrCreate(@NonNull final AccountDimension dimension)
 		return AccountId.ofRepoId(vc.getC_ValidCombination_ID());
 	}	// get
 
+
+	@Override
+	public List<AccountId> retrieveAccountsForTimeFrame(@NonNull final AcctSchemaId acctSchemaId, @NonNull final Timestamp dateAcctFrom, @NonNull final Timestamp dateAcctTo)
+	{
+
+		final List<Map<String, Object>> listDistinct = queryBL
+				.createQueryBuilder(I_Fact_Acct.class)
+				.addEqualsFilter(I_Fact_Acct.COLUMNNAME_C_AcctSchema_ID, acctSchemaId)
+				.addBetweenFilter(I_Fact_Acct.COLUMNNAME_DateAcct, dateAcctFrom,dateAcctTo)
+				.create()
+				.listDistinct(I_Fact_Acct.COLUMNNAME_Account_ID);
```

Comment:
pls use org.compiere.model.IQuery#listDistinct(java.lang.String, java.lang.Class<AT>)

i.e.
```java
.listDistinct(I_Fact_Acct.COLUMNNAME_Account_ID, AccountId.class);
```

Reply:
Done, just is ElementValueId